modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useRef } from 'react';
+import { Box } from '@elementor/ui';
+import PropTypes from 'prop-types';
+
+const THUMBNAIL_SIZE = 64;
+
+export const Thumbnail = ( props ) => {
+	const previewRef = useRef( null );
+
+	useEffect( () => {
+		if ( previewRef.current ) {
+			const previewRoot = previewRef.current.querySelector( '*' );
```

Comment:
you can probably do this:
```suggestion
			const previewRoot = previewRef.current.firstElementChild;
```

but I'm not sure what I'm seeing